Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/ui/main/home/HomeData.kt

```diff
@@ -35,6 +35,20 @@ object HomeData {
 
     object Tracing {
 
+        val LOW_RISK_ITEM_NO_ENCOUNTERS_WO_INSTALL_TIME = LowRiskCard.Item(
```

Comment:
Maybe type out `Without` instead of only `WO` to make it easier to understand (in several places)